plugins/team-harness/skills/update/scripts/converge.py

```
@@ -0,0 +1,892 @@
+def close_process_streams(process: subprocess.Popen[bytes], streams: selectors.BaseSelector) -> None:
+    streams.close()
+    if process.stdout is not None:
+        process.stdout.close()
+    if process.stderr is not None:
+        process.stderr.close()
+
+
+def run_native(argv: list[str]) -> str:
+    process: subprocess.Popen[bytes] | None = None
+    try:
+        process = subprocess.Popen(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)